components/base/Header.vue

```diff
@@ -0,0 +1,33 @@
+<script lang="ts" setup>
+import { ElMenu, ElSwitch } from "element-plus";
+import { Moon, Sunny } from "@element-plus/icons-vue";
+import { useDark, useToggle } from "@vueuse/core";
+
+const isDark = useDark();
+const toggleDark = useToggle(isDark);
+const theme = ref();
+theme.value = isDark.value;
+
+</script>
+
+<template>
+  <el-menu
+    mode="horizontal"
+  >
+    <div>
+      <nuxt-link to="/" class="pt-2 pb-2">
+        <img class="h-20" src="/assets/images/logo.png" alt="logo">
+      </nuxt-link>
+    </div>
+    <div class="flex items-center" @click="toggleDark()">
```

Comment:
looks weird 

<div class="flex items-center" @click="toggleDark()">
      <el-switch
        v-model="theme"

you should have a single source of truth